tests/compile/test_compile_ranges.py

```diff
@@ -49,47 +46,56 @@ def run_model(vllm_config: VllmConfig, model: nn.Module, batch_sizes: list[int])
             model(torch.randn(batch_size, MLP_SIZE).cuda())
 
 
-class PostGradPassManagerCheckRanges(CustomGraphPass):
-    def __init__(self, ranges: list[tuple[int, int]]):
+class PostGradPassManagerCheckRanges(InductorPass):
+    def __init__(self, ranges: list[Range]):
         self.ranges = ranges
+        self.num_calls = 0
 
     def __call__(self, graph: fx.Graph):
         compile_range = get_pass_context().compile_range
         assert compile_range in self.ranges, (
             f"Compile range {compile_range} not in {self.ranges}"
         )
+        self.num_calls += 1
```

Comment:
Can we count per-range to make sure it's not just one range called over and over?